python/ray/tune/trial.py

```diff
@@ -301,7 +301,7 @@ def __init__(self,
         if trial_dirname_creator:
             self.custom_dirname = trial_dirname_creator(self)
             if os.path.sep in self.custom_dirname:
-                raise ValueError(f"Trial dirname must not contain '/'. "
+                raise ValueError("Trial dirname must not contain '/'. "
                                  "Got {self.custom_dirname}")
```

Comment:
```suggestion
                                 f"Got {self.custom_dirname}")
```

Reply:
(mind i piggyback this fix)

Reply:
Good catch! I fixed another one of these, thanks for catching this one :D